src/apis/emscripten/mod.rs

```diff
@@ -44,11 +46,10 @@ fn dynamictop_ptr(static_bump: u32) -> u32 {
     static_bump + DYNAMICTOP_PTR_DIFF
 }
 
-// fn static_alloc(size: usize, static_top: &mut size) -> usize {
-//     let ret = *static_top;
-//     *static_top = (*static_top + size + 15) & (-16 as usize);
-//     ret
-// }
+
+pub fn statictop(static_bump: u32) -> u32 {
+    GLOBAL_BASE + 5520
+}
```

Comment:
I'm not sure if this is a good idea. Why did you make it fixed?

Reply:
I should probably remove it; this task doesn't depend on it. I was tryings things from the emscripten side.